NUKnightLab/storyline
# fix: Adding catch for errors in data parsing and messaging

To test:
Change the entries in `data_column_name`, `datetime_column_name`, `url` fields. You should see a message in the storyline container indicating what the error may be.
